rclpy/src/rclpy/_rclpy.c

```diff
@@ -42,6 +42,8 @@
 #include <rmw/validate_node_name.h>
 #include <rosidl_runtime_c/message_type_support_struct.h>
 
+#include "rcutils/macros.h"
```

Comment:
This should be moved up to the other `rcutils` import directives (in alphabetical order) and using `<...>` instead of quotes.